src/scaffoldmaker/meshtypes/meshtype_3d_wholebody1.py

```diff
@@ -0,0 +1,435 @@
+                        result4 = coordinates.setNodeParameters(fieldcache, -1, Node.VALUE_LABEL_VALUE, 1, x)
+                        result4 = coordinates.setNodeParameters(fieldcache, -1, Node.VALUE_LABEL_D_DS1, 1, d1)
+                        result4 = coordinates.setNodeParameters(fieldcache, -1, Node.VALUE_LABEL_D_DS2, 1, d2)
+                        result4 = coordinates.setNodeParameters(fieldcache, -1, valueLabel, 1, d3)
+                        result5 = coordinates.setNodeParameters(fieldcache, -1, valueLabel, 2, d3)
+                remapEftNodeValueLabelsVersion(eft, localNodeIndexes, [valueLabel], 2)
+                result1 = elementtemplate.defineField(coordinates, -1, eft)
+                result2 = element.merge(elementtemplate)
+                result3 = element.setNodesByIdentifier(eft, nodeIds)
+                element = elementIter.next()
+
+        # Annotation fiducial point
+        markerGroup = findOrCreateFieldGroup(fieldmodule, "marker")
+        markerName = findOrCreateFieldStoredString(fieldmodule, name="marker_name")
+        markerLocation = findOrCreateFieldStoredMeshLocation(fieldmodule, mesh, name="marker_location")
+        nodes = fieldmodule.findNodesetByFieldDomainType(Field.DOMAIN_TYPE_NODES)
+        markerPoints = findOrCreateFieldNodeGroup(markerGroup, nodes).getNodesetGroup()
+        markerTemplateInternal = nodes.createNodetemplate()
+        markerTemplateInternal.defineField(markerName)
+        markerTemplateInternal.defineField(markerLocation)
+        # Apex annotation point
+        middleLeft = elementsCountAcrossMinor//2
+        topElem = elementsCountAcrossMajor - 1
+        middleRight = middleLeft - 1
+        neckFirstElem = elementsCountAlongAbdomen+elementsCountAlongThorax
+        thoraxFirstElem = elementsCountAlongAbdomen
+        middleDown = elementsCountAcrossMajor//2 - 1
+        elementIdentifiers = [elementId[1][0][middleLeft], elementId[1][topElem][middleLeft], elementId[neckFirstElem][0][middleRight], elementId[neckFirstElem][topElem][middleRight],
+                elementId[1][0][middleLeft], elementId[1][topElem][middleLeft], elementId[neckFirstElem][0][middleRight], elementId[neckFirstElem][topElem][middleRight],
+                elementId[thoraxFirstElem][middleDown][middleRight], elementId[thoraxFirstElem + 1][middleDown][middleRight], elementId[thoraxFirstElem + 1][middleDown][middleRight]]
+        markerNames = ['left hip joint', 'right hip joint', 'left shoulder joint', 'right shoulder joint',
+                       'along left femur', 'along right femur', 'along left humerus', 'along right humerus',
+                       'heart apex', 'atrial base', 'aorta']
+        locations = [[0.8, 0.5, 0.5], [0.2, 0.5, 0.5], [0.8, 0.5, 0.5], [0.2, 0.5, 0.5],
+                     [0.2, 0.99, 0.5], [0.8, 0.99, 0.5], [0.5, 0.0, 0.5], [0.5, 0.0, 0.5],
+                     [0.5, 0.5, 0.0], [0.7, 0.5, 0.0], [0.7, 0.5, 0.2]]
```

Comment:
Use a structure which groups the corresponding name, element and xi locations otherwise you create a maintenance hazard where someone can mismatch them. I suggest using a list of dict:
```
markerPoints = [
    { "name" : "left hip joint", "elementId" : elementId[1][0][middleLeft], "xi" : [0.8, 0.5, 0.5] },
    { "name" : "right hip joint", "elementId" : elementId[1][topElem][middleLeft], "xi" : [0.2, 0.5, 0.5] },
    # etc.
    ]
```
Later you can refer to them as ``markerPoints[i]["name"], markerPoints[i]["elementId"], markerPoints[i]["xi"]``.

Reply:
Thanks for the suggestion, it makes total sense to use this.